ipatests/prci_definitions/nightly_latest.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
         git_refspec: '{git_refspec}'
         template: &ci-master-latest
           name: freeipa/ci-master-f32
-          version: 0.0.4
+          version: 0.0.5
```

Comment:
hi @netoarmando I expect some of the tests in the nightlies to fail with this change.